### DISCUSSION TITLE
How to ensure the onboarding commit message starts with an upper case letter

### DISCUSSION THREAD

**Discussion by JimFawkes** [renovatebot/renovate#28151] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

When I run renovate, an onboarding PR is created as expected. However it does not follow the expected format even though I explicitly define an onboarding message.

The commit message I expect:
```
chore(dependencies): Onboard renovate Bot by adding renovate.json
```

but the one I get is:
```
chore(dependencies): onboard renovate Bot by adding renovate.json
```

So I know that the message can have upper case chars and my custom message is used however the initial letter is still lower cased. How can I change this?

_The reason why I care is because we have a regex that blocks all commits pushed to any of our repos which do not follow the correct format, which in our case requires the initial letter of the message to be upper-case._

Here is my config.js:
```javascript
module.exports = {
  platform: "gitlab",
  endpoint: "https://some-url/api/v4",
  extends: [
    ":semanticCommits",
    ":separateMajorReleases",
    ":combinePatchMinorReleases",
    ":ignoreUnstable",
    ":prImmediately",
    ":semanticPrefixFixDepsChoreOthers",
    ":updateNotScheduled",
    ":automergeDisabled",
    ":ignoreModulesAndTests",
    "regexManagers:dockerfileVersions",
  ],
  branchPrefix: 'feature/',
  rangeStrategy: 'pin',
  semanticCommitScope: 'dependencies',
  username: 'renovate-bot',
  commitMessageLowerCase: 'never',
  gitAuthor: 'Renovate Bot <bot@renovateapp.com>',
  labels: ["renovate"],
  pinDigests: true,
  onboarding: true,
  onboardingBranch: 'feature/onboard-renovate',
  onboardingCommitMessage: 'Onboard renovate Bot by adding renovate.json',

  repositories: [
    'some-repo'
  ],

};
```
I am using renovate v37.270.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Possible duplicate of https://github.com/renovatebot/renovate/issues/25662

**Reply by JimFawkes** [reply]
Jipp, I think you are right! Thanks for the hint, somehow I missed that one earlier.
